src/core/Table/filters/FilterWrapper.tsx

```diff
@@ -0,0 +1,75 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import React from 'react';
+import '@itwin/itwinui-css/css/table.css';
+import Popover from '../../utils/Popover';
+import { useTheme } from '../../utils/hooks/useTheme';
+import { ColumnInstance, HeaderGroup } from 'react-table';
+import { TableFilterValue } from './types';
+
+export type FilterWrapperProps<T extends Record<string, unknown>> = {
+  column: HeaderGroup<T>;
+  allColumns: ColumnInstance<T>[];
```

Comment:
`allColumns` only needed for `length`? (which itself seems only needed for changing placement)

Reply:
Yup but if I remove that placement thing, it won't be needed.